lib/horizon/operation/invoke_host_function.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Stellar.Horizon.Operation.InvokeHostFunction do
+  @moduledoc """
+  Represents a `InvokeHostFunction` operation from Horizon API.
+  """
+
+  @behaviour Stellar.Horizon.Resource
+
+  alias Stellar.Horizon.Mapping
+
+  @type t :: %__MODULE__{
+          function: String.t(),
+          parameters: list(map()),
+          address: String.t(),
+          salt: String.t(),
+          asset_balance_changes: list()
+        }
```

Comment:
I've noticed that `parameters` and `asset_balance_changes` can be `nil`. So we should include `nil` in the typespec for these 2 fields.

Look at these examples: 
https://horizon-futurenet.stellar.org/operations/1766807811657729
https://horizon-futurenet.stellar.org/operations/1766803516690433



Also, for the `asset_balance_changes` field, we should be a bit more specific and write `list(map())`.